pages/transactional-email/how-to/add-dmarc-record.mdx

```diff
@@ -24,7 +24,7 @@ To complete the actions presented below, you must have:
 - A Scaleway account logged into the [console](https://console.scaleway.com/)
 - A domain name
 - [Configured your domain with Transactional Email](/transactional-email/how-to/configure-domain-with-transactional-email/).
-- [Owner](/iam/concepts/#owner) status of the Organization in which the actions will be performed, or be an IAM user with the [necessary permissions](/iam/concepts/#permission)
+- [Owner](/iam/concepts/#owner) status of the Organization in which the actions will be performed, or be an IAM user with the [necessary permissions](/iam/concepts/#permission). To follow this procedure you need `DomainsDNSFullAccess` and `TransactionalEmailDomainFullAccess` permissions.
```

Comment:
You only needs TransactionalEmailDomainFullAccess, the domain permission is only used if the domain in own by a SCW account, but it's not always the case